widget-store/src/utilities.ts

```diff
@@ -6,7 +6,8 @@ type KnexTransactionContext = TransactionContext<Knex>;
 
 export enum OrderStatus {
   PENDING = 0,
-  FULFILLED = 1,
+  DISPATCHED = 1,
+  PAID = 2,
```

Comment:
This is a new state, but it seems to briefly exist... after payment confirmation and before the kafka message is sent. So it seems to be mostly cosmetic in nature? Or did I miss something?

Reply:
If the kafka message failed it'd get left that way.
In a more serious (more complex / worse for demo) you wouldn't move on until you got a message from fulfillment.